sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessorOptions.cs

```diff
@@ -18,6 +18,7 @@ public class ServiceBusProcessorOptions
         /// from Queues or Subscriptions and queued locally, intended to help
         /// maximize throughput by allowing the processor to receive
         /// from a local cache rather than waiting on a service request.
+        /// The default value is 0.
```

Comment:
```suggestion
        /// The default value is 0.
```

This would be a good candidate to move to a sibling `<value>` tag for better visibility in the docs.